express-post-json/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+
+const app = express();
+
+const nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(Object.values(grades));
+  next();
```

Comment:
As we discussed in Q&A, this call is fine but not necessary for how we are using Express (and will continue using it).